HPE3PARPSToolkit-CLI.psm1

```diff
@@ -25450,7 +25474,7 @@ Function Set-3parRCopyTargetName
 		return "Error :  -TargetName is Mandate. "			
```

Comment:
Lots of places, you have given the return error message as "Error : <ParameterName> is Mandate" . 
Please replace these error return message to"Error: <ParameterName> is mandatory"

Reply:
Updated All.